docs/package.json

```diff
@@ -60,10 +60,12 @@
     "@types/node": "^14.14.31",
     "@types/react": "^17.0.2",
     "autoprefixer": "^10.2.4",
+    "dotenv-safe": "^8.2.0",
     "eslint-config-next": "12.0.4",
     "globby": "^13.1.1",
     "json5-loader": "^4.0.1",
     "postcss": "^8.2.6",
-    "tailwindcss": "^2.0.3"
+    "tailwindcss": "^2.0.3",
+    "ts-node": "^10.7.0"
```

Comment:
Should we consolidate our versions of ts-node at the root `devDependencies` since we use this in multiple packages?

Reply:
yep, I'll make a follow-up PR